### DISCUSSION TITLE
Web Interface Styles

### DISCUSSION THREAD

**Discussion by YeeJiaWei** [transmission/transmission#2657] [General]
Hi, is web client will have new Web Interface with better looking in recent release?

**Comment by ckerr** [comment]
"better looking" is subjective. It's not _that _ different from the previous interface IMO. The new web interface was more about improving the code, removing dependencies like jQuery, and about making it more mobile-friendly / touch-friendly.

You can try it out in a [nightly build](https://www.transmissionbt.com/download) and judge for yourself. There are also a few 3rd party web clients that you can try out if this one isn't to your liking
